cloudflare/provider.go

```diff
@@ -143,6 +143,7 @@ func Provider() *schema.Provider {
 			"cloudflare_custom_pages":                           resourceCloudflareCustomPages(),
 			"cloudflare_custom_ssl":                             resourceCloudflareCustomSsl(),
 			"cloudflare_device_posture_rule":                    resourceCloudflareDevicePostureRule(),
+			"cloudflare_device_policy_zone_certificates":        resourceCloudflareDevicePolicyZoneCertificates(),
```

Comment:
should this be `cloudflare_device_policy_certificates` and `zone_id` is just an optional attribute? will this ever live at the account level?

Reply:
Done.